rqd/rqd/cuerqd.py

```diff
@@ -31,8 +31,8 @@
 
 import grpc
 
-import rqd.compiled_proto.rqd_pb2
-import rqd.compiled_proto.rqd_pb2_grpc
+import cuebot.proto.rqd_pb2
+import cuebot.proto.rqd_pb2_grpc
```

Comment:
That's the root of my issue with cuebot.proto. This file is implementing the rqd interface, but this import tells me it belongs to cuebot. If we have an independent package opencue_proto.proto it makes sense that different modules will implement its interfaces, but if a single module owns the interface it tells me it also implements it fully.

Reply:
Have renamed it to `opencue_proto` now